src/mds/Server.cc

```diff
@@ -5419,11 +5427,95 @@ void Server::handle_client_setdirlayout(MDRequestRef& mdr)
 }
 
 // XATTRS
+std::pair<int, Server::JSONParsingError> Server::parse_layout_vxattr_json(
+  string name, string value, const OSDMap& osdmap, file_layout_t *layout)
+{
+  auto parse_pool = [&](std::string pool_name, int64_t pool_id) -> int64_t {
+    if (pool_name != "") {
+      int64_t _pool_id = osdmap.lookup_pg_pool_name(pool_name);
+      if (_pool_id < 0) {
+	dout(10) << __func__ << " unknown pool name:" << pool_name << dendl;
```

Comment:
```suggestion
	dout(10) << __func__ << ": unknown pool name:" << pool_name << dendl;
```

and elsewhere please

Reply:
ack; `:` now added before log string and elsewhere as well